djangocms_blog/models.py

```diff
@@ -287,7 +287,7 @@ class Meta:
         get_latest_by = 'date_published'
 
     def __str__(self):
-        return self.safe_translation_getter('title', any_language=True)
+        return self.safe_translation_getter('title', any_language=True, default="")
```

Comment:
I'd like more something like `Post (no translation)` (in a gettext marker). An empty string could make impossible to select the post or other actions

Reply:
I'll do the change and also edit the `tests.tests_models.ModelTest2.test_str_repr` to check this "no translation" case. Then pull request the `release/0.9.x` as you asked.

Reply:
PR #451

Reply:
Thanks, I'll close this